src/views/expensesgroup/addexpense/Dialogs/AddExpenseDialog.js

```diff
@@ -1,13 +1,19 @@
 // material ui
```

Comment:
Some improvements:

![image](https://user-images.githubusercontent.com/35969631/160106524-51b4cdee-955d-4faf-9ec2-539db59b2fdb.png)

Remove the button above, it's useless to have so many buttons to add expenses.
For each budget you can have a button to add the expenses RELATED with that budget.